RedditSharp/Multi/Multi.cs

```diff
@@ -0,0 +1,362 @@
+﻿using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Authentication;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace RedditSharp.Multi
+{
+    public class Multi
+    {
+        private Reddit Reddit { get; set; }
+        private IWebAgent WebAgent { get; set; }
+
+        #region Constant API URLs
+        private const string GetCurrentUserMultiUrl = "/api/multi/mine";
+        private const string GetPublicUserMultiUrl = "/api/multi/user/{0}";
+        private const string GetMultiPathUrl = "/api/multi/{0}";
+        private const string GetMultiDescriptionPathUrl = "/api/multi/{0}/description";
+        private const string GetMultiSubUrl = "/api/multi/{0}/r/{1}";
+        private const string PostMultiRenameUrl = "/api/multi/rename";
+        private const string PutSubMultiUrl = "/api/multi/{0}/r/{1}";
+        private const string CopyMultiUrl = "/api/multi/copy";
+
+        #endregion
+
+
+        public Multi(Reddit Reddit2, IWebAgent WebAgent2)
+        {
+            Reddit = Reddit2;
+            WebAgent = WebAgent2;
+        }
+
+        /// <summary>
+        /// Retrieve a list of the Multis belonging to the currently authenticated user
+        /// </summary>
+        /// <returns>A List of MultiData containing the authenticated user's Multis</returns>
+        public List<MultiData> GetCurrentUsersMultis()
+        {
+            var request = WebAgent.CreateGet(GetCurrentUserMultiUrl);
+            var response = request.GetResponse();
+            var json = JToken.Parse(WebAgent.GetResponseString(response.GetResponseStream()));
+            List<MultiData> results = new List<MultiData>();
+            for (int i = 0; i < json.Count();i++)
```

Comment:
Change this to a foreach loop? a normal for isn't exactly required here, just syntax sugar